holidays/countries/french_southern_territories.py

```diff
@@ -0,0 +1,50 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.countries.france import France
+
+
+class HolidaysTF(France):
+    """French Southern Territories holidays.
+    Since most islands doesn't have a permanent population,
+        the holidays are just the public ones from France
+    Alias of a French subdivision that is also officially assigned
+        its own country code in ISO 31661.
+
+    See:
```

Comment:
```suggestion

    Alias of a French subdivision that is also officially assigned
    its own country code in ISO 31661.

    !!! note "Note"
        Since most islands doesn't have a permanent population,
        the holidays are just the public ones from France.

    References:
```